src/logic/context/EditorContext.ts

```diff
@@ -0,0 +1,51 @@
+import {HotKeyAction} from "../../data/HotKeyAction";
+import {EditorModel} from "../../model/EditorModel";
+import {LabelType} from "../../data/LabelType";
+import {EditorData} from "../../data/EditorData";
+import {EditorActions} from "../actions/EditorActions";
+import {PolygonRenderEngine} from "../render/PolygonRenderEngine";
+import {EditorSelector} from "../../store/selectors/EditorSelector";
+import {store} from "../../index";
+import {updateActiveImageIndex} from "../../store/editor/actionCreators";
+
+export class EditorContext {
+    public static getActions(): HotKeyAction[] {
+        return [
```

Comment:
tutaj przy każdej zmianie kontekstu zawsze zwracasz nową tablicę. nie jest to pewnie jakiś mega błąd, ale potencjalne wycieki pamięci

Reply:
np: w konstruktorze tej klasy wywoływać tę metodę i zapisywać ją do property?